ui/components/Wallet/WalletActivityListItem.tsx

```diff
@@ -294,6 +296,9 @@ export default function WalletActivityListItem(props: Props): ReactElement {
             // For Infinite text in token approvals.
             text-transform: none;
           }
+          .name {
+            white-space: nowrap;
```

Comment:
Let's add top padding to align bottom text line of value and currency name
Before ![Screenshot 2022-09-07 at 11 59 22](https://user-images.githubusercontent.com/20949277/188850725-438d1e65-e5b0-4e9d-b91f-c0e8c04d0f80.png)
After ![Screenshot 2022-09-07 at 12 01 51](https://user-images.githubusercontent.com/20949277/188851121-e7dafc6d-d4d1-4225-b228-d05d1b25f0a4.png)


```suggestion
            white-space: nowrap;
            padding-top: 3px;
```